docs/resources.md

```diff
@@ -0,0 +1,41 @@
+# Adding Resource Files
```

Comment:
could we call that "assets" instead of "resources" so that it doesn't conflict with the "resources" field in the `package.json`?